CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ You can find its changes [documented below](#060---2020-06-01).
 - `Scale::from_scale` to `Scale::new`, and `Scale` methods `scale_x` / `scale_y` to `x` / `y`. ([#1042] by [@xStrom])
 - Major rework of keyboard event handling ([#1049] by [@raphlinus])
 - `Container::rounded` takes `KeyOrValue<f64>` instead of `f64`. ([#1054] by [@binomial0])
+- Re-export `druid_shell::Scalable` under `druid` namespace and remove `to_px`/`to_dp` helper fns on `Scale` ([#1075] by [@ForLoveOfCats])
```

Comment:
This should be split into two entries, one for the addition and one for the removal. Something like this:
### Added
- Re-export `druid_shell::Scalable` under `druid` namespace. ([#1075] by [@ForLoveOfCats])
### Removed
- `to_px`/`to_dp` helper functions on `Scale` in favor of those on `Scalable`. ([#1075] by [@ForLoveOfCats])